content/en/docs/concepts/rp.md

```diff
@@ -0,0 +1,553 @@
+---
+title: ResourcePlacement
+description: Concept about the ResourcePlacement API
+weight: 4
+---
+
+## Overview
+
+`ResourcePlacement` concept is used to dynamically select namespace scoped resources and control how they are propagated to all or a subset of the member clusters.
```

Comment:
Maybe add a motivation section on why we need this? Describe the difference between CRP and RP and the similarity?